Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM ubuntu:17.04
+FROM ubuntu:trusty
```

Comment:
Trusty is super old! 2014!

I use xenial myself, and then I don't need libicu hack.

Reply:
What's wrong with old? ;)

Truthfully, I picked `trusty` because it is what Travis uses, and therefore implies a little stability in my mind.

Reply:
As for `libicu`, I don't know enough about why it's required or how it should be installed to do anything but pick a compatible version!

Reply:
Fixed both in 4babfba